tests/plot/test_global_plot.py

```diff
@@ -0,0 +1,60 @@
+from unittest import TestCase
```

Comment:
As discussed this morning, the different tests you wrote are only testing if nothing bad happens while instantiating and plotting but does not test an expected behaviour. I opened an issue to explore whether this would be possible and interesting to set up: https://github.com/motleystate/moonstone/issues/11